apps/expo/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@babel/runtime": "^7.18.9",
     "@expo/config-plugins": "~7.2.2",
+    "@expo/webpack-config": "^19.0.0",
```

Comment:
trying to get dep graph to work better so added this because it has a webpack config option (as opposed to a metro one) but that didn't solve anything and seems it has problems resolving the config to how the app is actually packaged by expo/webpack (see `index.js` above)